CHANGELOG.md

```diff
@@ -1,6 +1,113 @@
+- The schema includes a number of new global/profile options, including: `transport.task_retry_initial_interval`, `transport.task_maximum_attempts`, `rmq.task_timeout` and `logging.aiopika_loglevel` ([#4583](https://github.com/aiidateam/aiida-core/pull/4583)).
+- The `cache_config.yml` has now also been **deprecated** and merged into the `config.json`, as part of the profile options.
+  This merge will be handled automatically, upon first load of the `config.json` using the new AiiDA version.
+
+In-line with these changes, the `verdi config` command has been refactored into separate commands, including `verdi config list`, `verdi config set`, `verdi config unset` and `verdi config caching`.
+
+See the [Configuring profile options](https://aiida.readthedocs.io/projects/aiida-core/en/v1.6.0/howto/installation.html#configuring-profile-options) and [Configuring caching](https://aiida.readthedocs.io/projects/aiida-core/en/v1.6.0/howto/run_codes.html#how-to-save-compute-time-with-caching) how-tos for more details.
+
+### Command-line additions and improvements 👌
+
+In addition to `verdi config`, numerous other new commands and options have been added to `verdi`:
+
+- **Deprecate** `verdi export` and `verdi import` commands and merge into `verdi archive` ([#4710](https://github.com/aiidateam/aiida-core/pull/4710))
+- Added `verdi group delete --delete-nodes`, to also delete the nodes in a group during its removal ([#4578](https://github.com/aiidateam/aiida-core/pull/4578)).
+- Improve `verdi group remove-nodes` command, to warn when requested nodes are not in the specified group ([#4728](https://github.com/aiidateam/aiida-core/pull/4728)).
+- Added `exception` to the projection mapping of `verdi process list`, for example to use in debugging as: `verdi process list -S excepted -P ctime pk exception` ([#4786](https://github.com/aiidateam/aiida-core/pull/4786)).
+- Added `verdi database summary` ([#4737](https://github.com/aiidateam/aiida-core/pull/4737)):
+  This prints a summary of the count of each entity and (optionally) the list of unique identifiers for some entities.
+- Improved `verdi process play` performance, by only querying for active processes with the `--all` flag ([#4671](https://github.com/aiidateam/aiida-core/pull/4671))
+- Added the `verdi database version` command ([#4613](https://github.com/aiidateam/aiida-core/pull/4613)):
+  This shows the schema generation and version of the database of the given profile, useful mostly for developers when debugging.
+- Improved `verdi node delete` performance ([#4575](https://github.com/aiidateam/aiida-core/pull/4575)):
+  Theed `verdi quicksetup --non-interactive`, to ensure it does not include any user prompts ([#4573](https://github.com/aiidateam/aiida-core/pull/4573))
```

Comment:
"Theed"? Is this a word I don't know or is this a cropping error?

Reply:
fixed regression